src/App.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import { withRouter, Route } from 'react-router-dom';
 
 import HomePage from './pages/HomePage';
+import NewslettersPage from './pages/NewslettersPage';
```

Comment:
Could you refactor the files for these pages to change the names:

- `NewslettersPage` to `NewsletterDetailPage`
- `AllNewslettersPage` to `NewslettersPage`